sympy/printing/latex.py

```diff
@@ -128,9 +128,10 @@ def latex_escape(s):
     We can't use verbatim easily with mathjax, so escaping is easier.
     Rules from https://tex.stackexchange.com/a/34586/41112.
     """
-    s = s.replace('\\', r'\textbackslash')
+    
     for c in '&%$#_{}':
         s = s.replace(c, '\\' + c)
+    s = s.replace('\\', r'\textbackslash')
```

Comment:
This isn't correct, this means that `_` is escaped to `\textbackslash_` instead of `\_`